storage/innobase/btr/btr0cur.cc

```diff
@@ -5137,7 +5137,7 @@ btr_cur_pessimistic_delete(
 
 			father_rec = btr_cur_get_rec(&father_cursor);
 			rtr_read_mbr(rec_get_nth_field(
-				father_rec, offsets, 0, &len), &father_mbr);
+				father_rec, offsets, 0, index, NULL, &len), &father_mbr);
```

Comment:
Rever this change. The format of spatial (R-tree) indexes is unaffected by the instant ADD COLUMN. Only the format of clustered index leaf pages is changed. And this is accessing a record in a spatial index.